slash/parallel/worker.py

```diff
@@ -67,8 +67,13 @@ def _stop_keepalive_thread(self):
             self._watchdog_thread.join()
 
     def start_execution(self, app, collected_tests):
-        if os.getpid() != os.getpgid(0):
-            os.setsid()
+        try:
+            if os.getpid() != os.getpgid(0):
+                os.setsid()
+        except AttributeError:
+            _logger.debug(
+                "os.getpgid and os.setsid() are not available on your platform"
```

Comment:
Change to `Skipping setsid for process since it is not available` or something like that